tests/test_metrics.py

```diff
@@ -57,12 +57,15 @@ def test_metrics(farm, runner):
     # parse metrics
     count_coco = []
     count_forward = []
+    count_wait_time = {}
     for metric in metrics:
         for sample in metric.samples:
             if sample.name == f"coco_dropped_request_total":
                 count_coco.append(sample)
             elif sample.name == f"coco_calls_total":
                 count_forward.append(sample)
+            elif sample.name == "coco_queue_wait_time_seconds_count":
```

Comment:
Why does the name end in `_count`?

Reply:
It's a series added by Prometheus for the Histogram type, and it tells you the total number of observations.

Reply:
makes sense, thanks.
This is what it produces:
```
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.005"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.01"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.025"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.05"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.075"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.1"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.25"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.5"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="0.75"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="1.0"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="2.5"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="5.0"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="7.5"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="10.0"} 2.0
coco_queue_wait_time_seconds_bucket{endpoint="status",le="+Inf"} 2.0
coco_queue_wait_time_seconds_count{endpoint="status"} 2.0
coco_queue_wait_time_seconds_sum{endpoint="status"} 0.001678466796875
```
Do we want to test any other values than the count?
Does it make sense that the buckets are cumulative and with upper bounds?